reverse_proxy/app/proxy.py

```
@@ -1,33 +1,89 @@
+
+async def scheduler_loop(back_writer):
+    """キューから最も優先度が高いデータを取り出し、サーバへ送る"""
+    while True:
+        priority, data, session = await scheduling_queue.get() # キューにデータが入るまで非同期で待機
+        try:
+            await asyncio.sleep(0.05)
+            tag = data[:4].decode(errors="ignore")
+            print(f"[Scheduler] send tag={tag} pri={priority:.6f} qsize={scheduling_queue.qsize()} from {session.client_info}")
+            back_writer.write(data) # データをバックに書き込む
+            await back_writer.drain() # 書き込みバッファが空になるまで待機
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)